fideslog/sdk/python/client.py

```diff
@@ -66,7 +66,7 @@ def __init__(
         self.developer_mode = developer_mode
         self.extra_data = extra_data or {}
 
-    def send(self, event: AnalyticsEvent) -> None:
+    def send(self, event: Union[AnalyticsEvent, UserRegistrationEvent]) -> None:
```

Comment:
I applaud the effort to keep the SDK simple, by allowing anyone call `.send()` with either an `AnalyticsEvent` or a `UserRegistrationEvent`, but it makes the downstream code (`__get_request_payload()` and `__send()`) difficult to grok. Try extracting portions of your rewrites of those functions into helper functions, and it'll make the code much more readable/maintainable.

Reply:
Good idea. One of the new helpers I added needed to become a static method because it didn't use `self`. Pylint didn't like that